app/models/middleware_messaging.rb

```diff
@@ -0,0 +1,12 @@
+class MiddlewareMessaging < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :middleware_server, :foreign_key => "server_id"
+  acts_as_miq_taggable
+  serialize :properties
+
+  include LiveMetricsMixin
```

Comment:
To simplify the PR, yes I guess it can be easier to add support for metrics in a following step.